contrib/plasma-pa/template.py

```diff
@@ -0,0 +1,41 @@
+pkgname = "plasma-pa"
+pkgver = "6.0.5"
+pkgrel = 0
+build_style = "cmake"
+# FIXME: only test, needs selenium-webdriver-at-spi-run
+make_check_args = ["-E", "applettest"]
+hostmakedepends = [
+    "cmake",
+    "extra-cmake-modules",
+    "gettext",
+    "ninja",
+    "pkgconf",
+]
+makedepends = [
+    "kcmutils-devel",
+    "kconfig-devel",
+    "kcoreaddons-devel",
+    "kdeclarative-devel",
+    "kdoctools-devel",
+    "kglobalaccel-devel",
+    "ki18n-devel",
+    "kstatusnotifieritem-devel",
+    "ksvg-devel",
+    "kwindowsystem-devel",
+    "libcanberra-devel",
+    "libplasma-devel",
+    "libpulse-devel",
+    "qt6-qtdeclarative-devel",
+]
+# sound-theme-freedesktop kirigami kirigami-addons kitemmodels?
+depends = [
+    "pipewire",
+]
```

Comment:
it doesn't depend on pipewire directly at all

Reply:
I could add it to `plasma-desktop-meta` instead I guess? I want people who install `plasma-desktop` to get a usable experience by default including audio similar to `base-desktop`

Reply:
yeah, should have those more toplevel in some meta thing